front/app/components/admin/ResourceList/index.tsx

```diff
@@ -28,6 +28,11 @@ export const Container = styled.div`
   padding-bottom: 10px;
   border-top: 1px solid ${colors.divider};
   transition: all ${timeout}ms cubic-bezier(0.165, 0.84, 0.44, 1);
+
+  ${({ disableNestedStyles }) =>
```

Comment:
Anything inside of the `Row` and `SortableRow` were a nightmare to style because of all of these nested styles, written in simpler and more naive times. Since we use this component in a huge amount of places, I couldn't go and fix it everywhere, so instead I added this prop which makes it behave a bit more normal.